components/eventing-controller/pkg/handlers/jetstream_test.go

```diff
@@ -961,196 +963,140 @@ func TestJSSubscriptionUsingCESDK(t *testing.T) {
 	require.NoError(t, jsBackend.DeleteSubscription(sub))
 }
 
-func TestJetStreamServerRestart_WithReconnectDisabled(t *testing.T) {
+// TestJetStream_ServerRestart tests that eventing works when NATS server is restarted
+// for scenarios involving the stream storage type and when reconnect attempts are exhausted or not.
+func TestJetStream_ServerRestart(t *testing.T) {
```

Comment:
Out of scope of this ticket, but we need to revisit the naming conventions for the Eventing tests in general.
I created a ticket for this here: https://github.com/kyma-project/kyma/issues/13685.